cmd/orb.go

```diff
@@ -203,6 +206,102 @@ Please note that at this time all orbs created in the registry are world-readabl
 	return orbCommand
 }
 
+func addOrbElementParametersToBuffer(buf *bytes.Buffer, orbElement api.OrbElement) error {
+	for parameterName, parameter := range orbElement.Parameters {
+		var err error
+		if parameter.Default != "" {
+			_, err = buf.WriteString(fmt.Sprintf("       - %s: %s (default: '%s')\n", parameterName, parameter.Type, parameter.Default))
```

Comment:
Having the indentation level be part of the string feels awkward, but I'm not sure if there's a better way...